app/javascript/components/server-components/DashboardPage.tsx

```diff
@@ -197,78 +246,115 @@ export const DashboardPage = ({
+                    name="Make an impression"
                     checked={getting_started_stats.customized_profile}
                     link={Routes.settings_profile_path()}
+                    IconComponent={CustomizeProfileIcon}
+                    description="Customize your profile."
                   />
                   <RadioItem
-                    name="Create your first product"
+                    name="Showtime"
                     checked={getting_started_stats.first_product}
                     link={Routes.new_product_path()}
+                    IconComponent={FirstProductIcon}
+                    description="Create your first product."
                   />
                   <RadioItem
-                    name="Get your first follower"
+                    name="Build your tribe"
                     checked={getting_started_stats.first_follower}
                     link={Routes.followers_path()}
+                    IconComponent={FirstFollowerIcon}
+                    description="Get your first follower."
                   />
                   <RadioItem
-                    name="Make your first sale"
+                    name="Cha-ching"
                     checked={getting_started_stats.first_sale}
                     link={Routes.sales_dashboard_path()}
+                    IconComponent={FirstSaleIcon}
+                    description="Make your first sale."
                   />
                   <RadioItem
-                    name="Get your first pay out"
+                    name="Money inbound"
                     checked={getting_started_stats.first_payout}
                     link={Routes.settings_payments_path()}
+                    IconComponent={FirstPayoutIcon}
+                    description="Get your first pay out."
                   />
                   <RadioItem
-                    name="Send out your first email blast"
+                    name="Making waves"
                     checked={getting_started_stats.first_email}
                     link={Routes.posts_path()}
+                    IconComponent={EmailBlastIcon}
+                    description="Send out your first email blast."
+                  />
+                  <RadioItem
+                    name="Smart move"
+                    checked={getting_started_stats.first_small_bets}
+                    link="https://dvassallo.gumroad.com/l/small-bets?layout=profile"
+                    IconComponent={SmallBetsIcon}
+                    description="Sign up for Small Bets."
                   />
                 </div>
               </div>
             )
           : null}
 
-        <ProductsTable sales={sales} />
+        {!getting_started_stats.first_product && loggedInUser?.policies.product.create ? <Greeter /> : null}
 
-        <div style={{ display: "grid", gap: "var(--spacer-4)" }}>
+        {!getting_started_stats.first_product && loggedInUser?.policies.product.create ? null : (
```

Comment:
I am going to assume that this isn't intentional. Will revert. The new condition makes it such that this is only visible when:

- the seller does NOT have any products, OR
- the current user does not have access to create products